torch/csrc/jit/register_c10_ops.cpp

```diff
@@ -10,7 +10,7 @@ at::Tensor unwrap(at::Tensor&& tensor) {
   if (tensor.requires_grad()) {
     throw std::runtime_error("Autograd not yet supported for c10 ops.");
   }
-  return torch::autograd::Variable(std::move(tensor)).data();
+  return std::move(tensor);
```

Comment:
It's okay to remove the `.data()` call here, because the check above this line throws error if `tensor` requires grad, so we know that `tensor` is guaranteed to not require grad at this point.

Reply:
I don't believe this move is necessary, see https://stackoverflow.com/questions/14856344/when-should-stdmove-be-used-on-a-function-return-value.